assets/dev/js/editor/document/save/behaviors/footer-saver.js

```diff
@@ -101,12 +101,23 @@ module.exports = class FooterSaver extends Marionette.Behavior {
 
 	addTooltip() {
 		// Create tooltip on controls
-		this.$el.find( '.tooltip-target' ).tipsy( {
-			// `n` for down, `s` for up
-			gravity: 's',
-			title() {
-				return this.getAttribute( 'data-tooltip' );
-			},
+		this.$el.find( '.tooltip-target' ).each( ( index, button ) => {
+			const $button = jQuery( button );
+			let offset = 0;
+
+			if ( $button.attr( 'id' ) === this.ui.buttonSaveOptions.attr( 'id' ) ) {
+				// The "Save Options" button's tooltip needs to be raised to the same height as other buttons
+				offset = 7;
+			}
+
+			$button.tipsy( {
+				// `n` for down, `s` for up
+				gravity: 's',
+				offset: offset,
+				title() {
+					return this.getAttribute( 'data-tooltip' );
+				},
+			} );
```

Comment:
Instead of that, try to move the `tooltip-target` class of the save button one level above

Reply:
If I do that, the tooltip is not centered to the arrow in the button. That is why I had to do all of this work